dev/drivers/scripts/cam/stats/jevs_cam_hireswarwmem2_snowfall_stats.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+#PBS -N jevs_cam_hireswarwmem2_snowfall_stats
+#PBS -j oe
+#PBS -S /bin/bash
+#PBS -q dev
+#PBS -A VERF-DEV
+#PBS -l walltime=10:00:00
+#PBS -l place=vscatter:exclhost,select=1:ncpus=128:ompthreads=1
+#PBS -l debug=true
+#PBS -V
+
+set -x
+export model=evs
+module reset
+export machine=WCOSS2
+
+# ECF Settings
+export RUN_ENVIR=nco
+export SENDECF=YES
+export SENDCOM=YES
+export KEEPDATA=YES
+export SENDDBN=YES
+export SENDDBN_NTC=
+export job=${PBS_JOBNAME:-jevs_cam_hireswarwmem2_snowfall_stats}
+export jobid=$job.${PBS_JOBID:-$$}
+export SITE=$(cat /etc/cluster_name)
+export USE_CFP=YES
+export nproc=128
+
+# General Verification Settings
+export NET="evs"
+export STEP="stats"
+export COMPONENT="cam"
+export RUN="atmos"
+export VERIF_CASE="snowfall"
+export MODELNAME="hireswarwmem2"
+
+# EVS Settings
+export HOMEevs="/lfs/h2/emc/vpppg/noscrub/$USER/EVS"
+export HOMEevs=${HOMEevs:-${PACKAGEROOT}/evs.${evs_ver}}
+export config=$HOMEevs/parm/evs_config/cam/config.evs.prod.${STEP}.${COMPONENT}.${RUN}.${VERIF_CASE}.${MODELNAME}
+
+# Load Modules
+source $HOMEevs/versions/run.ver
+source /usr/share/lmod/lmod/init/sh
+module reset
+source $HOMEevs/modulefiles/$COMPONENT/${COMPONENT}_${STEP}.sh
+export MET_bin_exec="bin"
```

Comment:
We can remove this line throughout all driver scripts now that the setting is contained within `machine.conf`.

Reply:
@ShelleyMelchior-NOAA I removed this line in all cam/precip, /grid2obs, and /snowfall driver scripts, and added the machine.conf as a config to all METplus runs to access its settings.